index.js

```diff
@@ -0,0 +1,158 @@
+// Manage Roles (id, name)
+const express = require('express');
+const helmet = require('helmet');
+const knex = require('knex');
+const students = require('./studentRoutes.js');
+
+// Knex CLI commands
+// knex is not installed globally so all commands are prepended with yarn or npx
+//
+// Migrations allow for you to define sets of schema changes so upgrading a database is a breeze.
+// yarn knex init                               // creates config file empty
+// yarn knex migrate:make roles_table           // makes a migration roles_table
+// yarn knex migrate:latest                     // updates to latest migration
+// yarn knex migrate:rollback                   // Rollback migration. rollback will revert to most recent batch. 
+// yarn knex migrate:make fix_roles_latinName   // makes a migration called fix_roles_latinName
+//
+// Seed files allow you to populate your database with test or seed data independent of your migration files.
+// yarn knex seed:make seed_name                // makes a seed file called seed_name
+// yarn knex seed:run                           // Runs seed files, populates you db with seed data
+
+
+// Knex configuration. Then, assign it to 'db', invoking the config through knex
+const knexConfig = {
+    client: 'sqlite3',
+    connection: {
+      filename: './data/lambda.sqlite3',
+    },
+    useNullAsDefault: true,             // needed for sqlite
+    // debug: true,                        // Useful for development
+};
+const db = knex(knexConfig);
+
+// Variables
+const port = process.env.PORT || 5000;
+const server = express();
+const errors = { // Dynamic error messaging based on sqlite codes
+    '1': 'We ran into an error.',
+    '4': 'Operation aborted',
+    '9': 'Operation aborted',
+    '19': 'Another record with that value exists, yo!'
+};
+
+// Middleware
+server.use(helmet());
+server.use(express.json());
+server.use('/students', students);
+
+
+// Endpoints
+// GET all cohorts
+server.get('/api/cohorts', async (req, res) => {
+    try {
+        const cohorts = await db('cohorts');
+        res.status(200).json(cohorts);
+    } catch (error) {
+        res.status(500).json(error);
+    }
+});
+
+
+// GET a cohort by id
+server.get('/api/cohorts/:id', async (req, res) => {
+    try {
+        const cohort = await db('cohorts')
+            .where({ id: req.params.id })
+            .first();
+        res.status(200).json(cohort);
+    } catch (error) {
+        res.status(500).json(error);
+    }
+});
+
+
+// GET students in cohort by cohort ID
+server.get('/api/cohorts/:id/students', async (req, res) => {
+    try {
+        const cohortStudents = await db('cohorts')
+            .innerJoin('students', `cohorts.id`, 'students.cohort_id')
```

Comment:
since it defaults to innerJoin, you could actually get away with just saying 'join' here